db/seeds/superkey_metadata.yml

```diff
@@ -55,11 +53,11 @@
         ]
       }
     substitutions:
-    - ACCOUNT: get_account
+      ACCOUNT: get_account
   - step: 3
     name: bind_role
     payload: {}
-    substitutions: []
+    substitutions: {}
```

Comment:
Just for my curiosity, why did substitutions change from an `[]` to a `{}`?

Reply:
Basically I [changed my schema](https://github.com/lindgrenj6/sources-superkey-worker/blob/03fc9fd558b1a23f4917d17b42179f40ce8ffea5/provider/types.go#L22) on the superkey side to be a `map[string]string` instead of an `[]map[string]string`, since each item was a single k/v pair to begin with. It makes more sense to iterate over k/v in the map than to iterate over an array of single k/v pairs.